scripts/local-install.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+## Example:
+##
+## local-install.sh sdk/js-client-sdk-common sdk/js-client-sdk && local-install.sh sdk/js-client-sdk test-apps/my-test-react-app
+##
+
+## (requires jq (`brew install jq`))
+
+if [[ "$1" = "" || "$2" = "" ]]; then
+  echo 'USAGE: ./local-install.sh <package-to-install-path> <consuming-app-path>'
+  ehco ''
+  echo ''
+  exit 1
+fi
+
+if [ ! -d "$1" ]; then
+  echo "$1 is not a directory"
+  exit 1
+fi
+
+if [ ! -f "$1/package.json" ]; then
+  echo "$1 is not an npm package"
+  exit 1
+fi
+
+if [ ! -d "$2" ]; then
+  echo "$2 is not a directory"
+  exit 1
+fi
+
+rm -rf ~/.yarn/berry/cache/\@eppo-*
+
+### Create local package via "npm pack"
+pushd "$1" > /dev/null
+rm -rf package pack.out pack.err && npm pack 2> pack.err > pack.out
```

Comment:
after using this script for a bit, would love for these intermediate files to be created in a temporary directory or something

Reply:
I can make that happen!

Reply:
updated!